nodes/src/nodes/eval_cobalt/evaluators/format_check.py

```
@@ -0,0 +1,203 @@
+        output: The LLM-generated output to evaluate.
+        expected_format: The expected format type. One of: prose, list, code, json.
+        threshold: Minimum score to pass (default 0.5).
+
+    Returns:
+        A dict with keys: score (float 0-1), passed (bool), reasoning (str).
+
+    Raises:
+        ValueError: If expected_format is not one of the supported types.
+    """
+    if expected_format not in _FORMAT_CHECKERS:
+        raise ValueError(f'Unsupported format type: {expected_format!r}. Supported: {list(_FORMAT_CHECKERS.keys())}')
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)